### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by nouman17610**
I implemented the following

- Header with the main banner
- Biography Area
- Footer with link to wikipedia

### CODE REVIEW

**Comment by nouman17610** [comment]
The excellent effort so far with HTML and CSS. There are some changes required
Pull request has no Linters. Linters must be added to know the quality of the code.

**Comment by nouman17610** [comment]
Home page by nouman17610 · Pull Request #1 · nouman17610_project-code-review

Within the body tag, the first tag should be the header followed by the main tag. Best practices recommend the top menu area within the header and the remaining area within main. Wthin main tag each area needs to be in the section tags.

**Comment by nouman17610** [comment]
tribute-page freecodecamp rocks
Dr  Norman Borlaug

Your design needs to show a similar level of spacing as available in the sample design. Your background color doesn't match to the sample design. Please use same level of spacing, font sizes, and image size.

**Comment by nouman17610** [comment]
There is a good PR description and satisfactory commit messages.

**Comment by nouman17610** [comment]
Home page by nouman17610 · Pull Request #1 · nouman17610_project-code-review

It is best practice to code for the mobile screen first and then the web page. Please add a media query for the minimum screen size of 375 pixels for mobile screens and 768 pixels for desktop screens.

**Comment by nouman17610** [comment]
The read-me file doesn't contain any information about the project features. Please add a professional template and description relevant to the project.

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @nouman17610 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- Comments are Polite :heavy_check_mark: 
- Comments include a why :heavy_check_mark: 

 ## Required Changes ♻️
- [ ] Great job so far @nouman17610  There are just a few changes you need to make and you'll be on your way to the next Milestone :+1: 
- [ ] I noticed that you didn't use GitHub inline comments leave at least one inline comment below just as mentioned in the project requirements :+1: Kindly take a look at [this guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) and apply the concepts discussed here to your code review. :100: 
- [ ] Kindly Remember to be polite, and always give reasons for the requested change :wink: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+               <li><strong>2005</strong> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds."</li> 
+               <li><strong>2009</strong> - dies at the age of 95.</li> 
+               <q id="quote">
+                Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."
+
+                -- Indian Prime Minister Manmohan Singh
+               </q>
+            </ul>
+        </section>
+        <footer id="footer">
```
Comment (nouman17610) [inline index.html:42]: It is great you have added footer tag but it should not be within the main tag. It should be after the closing of the main tag. Please add a footer tag after the main tag not within the main tag.

Comment (wayungi) [reply index.html:42]: - [ ] Kindly help the student understand why the footer tag should be outside the main tag. In doing so, you help the student to learn. That's what code reviews are for. Kindly give a reason why the footer should not be n the main tag 👍🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @nouman17610 , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] Good attempt at using inline comments ✔️ 
- [ ] At least one comment given using GitHub inline features 👍🏾 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] You need at least 2 inline comments to move  to the next step. 👍🏾 
- [ ] Keep a polite tone and give explanations for every change request 👍🏾 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (nouman17610) [reply index.html:42]: The reasons for adding a footer tag after the main tag are:

- It helps in SEO-friendliness and accessibility of the web-page
- The main tag contains content of a specific page but footer contains the content that may be repeated in multiple pages. It is common practice to add a footer tag after the main tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+               <li><strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed, creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.</li> 
+               <li><strong>1970</strong> - receives the Nobel Peace Prize</li> 
+               <li><strong>1983</strong> - helps seven African countries dramatically increase their maize and sorghum yields</li> 
+               <li><strong>1984</strong> - becomes a distinguished professor at Texas A&M University</li> 
+               <li><strong>2005</strong> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds."</li> 
+               <li><strong>2009</strong> - dies at the age of 95.</li> 
+               <q id="quote">
```
Comment (nouman17610) [inline index.html:35]: It is good to you have an id to identify a specific tag but you haven't used any class attribute to any of the tags. Please it is important to use the id attribute for the tag that is unique in the web page. 
Please class attribute where the tag may repeat and please use the id where the tag is unique on the web page.

**Comment by nahnahdev** [comment]
## Project Status: APPROVED :green_circle: 
Hi @nouman17610 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 
- Good use of GitHub inline comments :heavy_check_mark: 
- Reasons added for suggested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**